docker/_include.sh

```diff
@@ -10,7 +10,6 @@ XDG_CACHE_HOME=${XDG_CACHE_HOME:-$HOME/.cache}
 PROJECT_NAME="commcarehq"
 
 function web_runner() {
-    sudo \
-        env DOCKER_DATA_HOME=$DOCKER_DATA_HOME XDG_CACHE_HOME=$XDG_CACHE_HOME\
-        docker-compose -f $DOCKER_DIR/compose/docker-compose-hq.yml -p $PROJECT_NAME $@
+    env DOCKER_DATA_HOME=$DOCKER_DATA_HOME XDG_CACHE_HOME=$XDG_CACHE_HOME\
+        sudo docker-compose -f $DOCKER_DIR/compose/docker-compose-hq.yml -p $PROJECT_NAME $@
```

Comment:
The problem with doing it this way is that the env vars don't get forward to the sudo session. The other option is to add yourself to the 'docker' group and then remove the `sudo` command altogether. Unfortunately this doesn't work with Travis (which requires sudo). We could have a separate script for Travis that uses sudo?